tests/sut/sut.go

```diff
@@ -31,9 +31,10 @@ const (
 
 	TimeoutRawDiskTest = 600 // Timeout to connect for recovery_raw_disk_test
 
-	Ext2 = "ext2"
-	Ext3 = "ext3"
-	Ext4 = "ext4"
+	Ext2         = "ext2"
```

Comment:
mmm damn, we have been moving this file out from the repo with #1368 so we can share it with different projects, maybe can put those changes directly in the [testhelper](https://github.com/rancher-sandbox/ele-testhelpers/blob/main/vm/sut.go) repo?

Reply:
sure, let me first use this to get the actual log of the upgrade tests because somehow is resetting to a wrong version and I have no visibility over that